data-prepper-plugins/otlp-sink/build.gradle

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id 'java-library'
+    id 'jacoco'
+}
+
+jacoco {
```

Comment:
Let's use the same tool version for JaCoCo from the parent. So remove these three lines.